easybuild/easyconfigs/x/xmlf90/xmlf90-1.5.3-foss-2016b.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = 'ConfigureMake'
+
+name = 'xmlf90'
+version = '1.5.3'
+
+homepage = 'https://launchpad.net/xmlf90'
+description = """xmlf90 is a basic XML parsing library written in Fortran."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['https://launchpad.net/xmlf90/trunk/1.5/+download/']
+checksums = ['bf6b8fd83cd1c7fae6f9451a5babde82']
```

Comment:
please use `SHA256` checksum, avoid hardcoding `1.5`, use `%(version_major_minor)s` instead

Reply:
Done and took note for the future.